backend/src/controllers/reports/Tab_18_rpt_P_StatusSummary.js

```diff
@@ -18,7 +18,6 @@ controller.getReport = getReport;
  * @returns {object}
  */
 controller.Tab_18_rpt_P_StatusSummary = async (request, reply) => {
-  controller.userRequires(request, "PMO-Reports-Capability", reply);
```

Comment:
needs this role added to the route config.role

Reply:
ah ok I'll go edit these old routes then tomorrow

Reply:
It's in the route already, thanks to you!